deployer/src/handlers/error.rs

```diff
@@ -0,0 +1,59 @@
+use std::error::Error as StdError;
+
+use axum::http::{header, HeaderValue, StatusCode};
+use axum::response::{IntoResponse, Response};
+use axum::Json;
+
+use serde::{ser::SerializeMap, Serialize};
+use shuttle_common::models::error::ApiError;
+use tracing::error;
+use utoipa::ToSchema;
+
+#[derive(thiserror::Error, Debug, ToSchema)]
+pub enum Error {
+    #[error("Streaming error: {0}")]
+    Streaming(#[from] axum::Error),
+    #[error("{0}, try running `cargo shuttle deploy`")]
+    NotFound(String),
```

Comment:
Is this correct? I think we'll no longer have a not found :man_shrugging:

Reply:
Removed it. Done.